test/DynamicDependency/data/PushNotificationsLongRunningTask.Msix/appxmanifest.xml

```diff
@@ -0,0 +1,69 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You'll want to move this directory to somewhere under your feature e.g. `test\PushNotifications\data\PushNotificationLongRunningTask.msix`

General rule for the repo, keep a feature's test collateral with the feature's tests
Thus `test\PushNotifications\...something...`
We won't have a `test\packages\...everything into the pool...and watch the it turn into a complex cesspool...`

Yes, we do have some cases of that cesspool anti-pattern. They're on the punchlist to cleanup, after other priority work's addressed

It may not seem so bad now, but the project's young with few developers (relatively speaking). Scale up going forward to 10x and 100x the developers and features and you can imagine how crazed it would get... That's why we're trying to push good structure now. Start as you mean to go on...

Reply:
Makes sense. I'll move this to a separate PushNotifications folder.